linux_os/guide/system/permissions/files/permissions_important_account_files/file_permissions_backup_etc_shadow/rule.yml

```diff
@@ -45,6 +45,8 @@ template:
         filemode@debian9: '0640'
         filemode@debian10: '0640'
         filemode@debian11: '0640'
+        filemode@sle12: '640'
+        filemode@sle15: '640'
```

Comment:
Is there any particular reason why not to have the left zero here as other distributions? The rest of the rule will render descriptions and etc with `0640` anyway.

Reply:
6.1.6 in the sle15 benchmark recommends 640 or more restrictive. So I guess both would be fine. If you want to have it on 000 for consistency , I'll update it. And tbh, I've barely seen a system where this file even existed.

Reply:
I have removed the changes. Better safe (000) than sorry (640).

Reply:
I would leave it the way it is, to keep it uniform and because both 640 and 000 are correct implementations of 6.1.6.

If it proofs to be too restrictive, I or someone else will update the value later.